modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/ZonePartitionReplicaListener.java

```diff
@@ -58,15 +58,18 @@ public class ZonePartitionReplicaListener implements ReplicaListener {
     private static final IgniteLogger LOG = Loggers.forClass(ZonePartitionReplicaListener.class);
 
     // TODO: https://issues.apache.org/jira/browse/IGNITE-22624 await for the table replica listener if needed.
-    private final Map<TablePartitionId, ReplicaListener> replicas = new ConcurrentHashMap<>();
+    private final Map<Integer, ReplicaListener> replicas = new ConcurrentHashMap<>();
```

Comment:
Please add our comment comment 
// tableId -> tableProcessor
or similar. Otherwise it'll require more time to understand that Integer is tableId and ReplciaListener is table listener.